specification/gedcom-3-structures-1-organization.md

```diff
@@ -153,7 +155,7 @@ n HEAD                                     {1:1}  g7:HEAD
   +1 SUBM @<XREF:SUBM>@                    {0:1}  g7:SUBM
   +1 COPR <Text>                           {0:1}  g7:COPR
   +1 LANG <Language>                       {0:1}  g7:HEAD-LANG
-  +1 PLAC                                  {0:1}  g7:HEAD-PLAC
+  +1 PLAC                                  {1:1}  g7:HEAD-PLAC
```

Comment:
Discussion in GEDCOM Steering Committee:
* Requiring this would not allowing supporting cases where one doesn't know the names of the jurisdiction labels, such as "Champaign" where the source record doesn't specify whether it's Champaign the city in Illinois or the county in Illinois.
* The committee is considering the possibility of deprecating the use of `HEAD`.`PLAC`.`FORM` all together, but wants more input on its use.

Reply:
One of the programs I work with, when creating a new GEDCOM, it outputs the "default" jurisdictions there, depending on the language of the GEDCOM. So for French it outputs "Lieudit, Commune, Code_INSEE, Code_Postal, Département, Région, Pays", for Dutch it outputs "Gehucht, Plaats, Postcode, Provincie, Staat, Land" (So French or Dutch jurisdictions, which, in my opinion, is wrong, and should have been english in all cases, as other programs might not understand jurisdictions in all languages)  

That also happens when it imports a GEDCOM that has no jurisdictions in the HEAD.
So it IS used.

But as it is mandatory not all programs have that in the HEAD, causing lots of trouble when another program tries to interpret places entered in all kind of forms, so in an unstructured way.  
It often means, that after importing their file in another program, users have to go over their places by hand to be able to get the correct map coordinates.  
Thats the reason I made it {1:1}  
Coming from a widely used Dutch program that has nothing at all for PLAC.FORM, not in their HEAD, nor in the file itself I have seen many users struggling with that when they want to import somewhere else.

I always interpreted HEAD.PLAC.FORM as the base rule for jurisdictions. The default jurisdictions so to speak.
Defining a **default** set of jurisdictions, isnt that separate of how a certain place inside a GEDCOM is defined?

As you say about "Champaign" a user might not know exactly what it is. But that is only one place somewhere in the GEDCOM. Most other places might easily fit in the default jurisdictions.  
In case of SPLAC, Champaign would only have the Champaign SPLAC and no further link to a parent SPLAC, so i think that would still be a problem then.

So if those "defaults" would be deprecated, how to replace them?
Online systems, often used by genealogical programs to grab the coords for a place, use a certain amount of "jurisdictions" to be able to find that place in their database. Just the placename is often not sufficient.

These were a couple of my thoughts when I changed this to {1:1}  
Do you rather  want me to change it back?